packages/git-utils/src/exec.js

```diff
@@ -9,8 +9,15 @@ const pathExists = require('path-exists')
 // Fires the `git` binary. Memoized.
 const mGit = function (args, cwd) {
   const cwdA = safeGetCwd(cwd)
-  const { stdout } = execa.sync('git', args, { cwd: cwdA })
-  return stdout
+  // The child process `error.message` includes stderr and stdout output which most of the times contains duplicate
+  // information. We rely on `error.shortMessage` instead.
```

Comment:
In this instance, wouldn't the git output be useful for debugging purpose?

Reply:
The message we're getting contains the exact same input twice. Sharing the snippet so you can see:
```md
## Git utils fails if no root

> Snapshot 1

    `␊
    ────────────────────────────────────────────────────────────────␊
      Netlify Build                                                 ␊
    ────────────────────────────────────────────────────────────────␊
    ␊
    > Version␊
      @netlify/build 1.0.0␊
    ␊
    > Flags␊
      debug: true␊
      repositoryRoot: /tmp-dir␊
      testOpts:␊
        pluginsListUrl: test␊
        silentLingeringProcesses: true␊
    ␊
    > Current directory␊
      /tmp-dir␊
    ␊
    > Config file␊
      /tmp-dir/netlify.toml␊
    ␊
    > Resolved config␊
      build:␊
        publish: /tmp-dir␊
        publishOrigin: default␊
      plugins:␊
        - inputs: {}␊
          origin: config␊
          package: ./plugin␊
    ␊
    > Context␊
      production␊
    ␊
    > Loading plugins␊
       - ./plugin from netlify.toml␊
    ␊
    ────────────────────────────────────────────────────────────────␊
      1. onPreBuild command from ./plugin                           ␊
    ────────────────────────────────────────────────────────────────␊
    ␊
    ␊
    ────────────────────────────────────────────────────────────────␊
      Plugin "./plugin" internal error                              ␊
    ────────────────────────────────────────────────────────────────␊
    ␊
      Error message␊
      Error: Invalid head commit HEAD␊
      Command failed with exit code 128: git rev-parse HEAD␊
      fatal: not a git repository (or any of the parent directories): .git␊
      fatal: not a git repository (or any of the parent directories): .git␊
    ␊
      Error location␊
      In "onPreBuild" event in "./plugin" from netlify.toml␊
    STACK TRACE␊
    ␊
      Resolved config␊
      build:␊
        publish: /tmp-dir␊
        publishOrigin: default␊
      plugins:␊
        - inputs: {}␊
          origin: config␊
          package: ./plugin`

```

My understanding is that in order to keep the current behaviour we'll need to rely on `shortMessage` as the error message has the same input contained in stdout/stderr?

Reply:
Oh, yes. Makes sense :+1: